validator/validation_entry.go

```diff
@@ -22,4 +23,9 @@ type ValidationInput struct {
 	DelayedMsg    []byte
 	StartState    GoGlobalState
 	DebugChain    bool
+
```

Comment:
Extra validation input for validation of Espresso integration functionality.

- The height is used for continuity check of blocks.
- The hotshot commitment is used to verify the integrity of transactions.
- The hotshot liveness is used to validate correct use of the escape hatch.